packages/core/src/App/initStore.js

```diff
@@ -30,11 +31,37 @@ const setStorageEvents = root_store => {
     });
 };
 
+const startPerformanceMetrics = (url_query_string, url_params) => {
+    // start the timer for signup
+    if (url_params.get('action') === 'signup') startPerformanceEventTimer('signup_time');
+
+    // start the timer for login
+    if (url_params.get('acct1')) startPerformanceEventTimer('login_time');
+
+    // start the timer for redirect from deriv.com
+    if (url_params.get('redirect_from') === 'deriv_com') {
+        startPerformanceEventTimer('redirect_from_deriv_com_time');
+
+        // remove 'redirect_from' query param
+        history.replaceState(
```

Comment:
Why do we need to remove the query param?

Reply:
@habib-deriv if we don't remove this query param then after the page refresh we'll define this as redirect from deriv.com again, but it's not true.